universe/common/api/dto/types.go

```diff
@@ -10,8 +10,8 @@ const (
 type FlyWithMeType string
```

Comment:
cosmetic nit: if you think it makes sense rename this to `FlyToMeType`

Reply:
changed - thanks